src/incubator/f5-bigip-ingress/README.md

```diff
@@ -1,12 +1,26 @@
-# alpha status
+# Helm Chart for Managing Ingress Resources with a BIG-IP Device
 
-This chart is in an alpha status. It can be used to deploy an ingress with on simple service. 
+This chart simplifies repeatable, versioned use of the [F5 BIG-IP Controller as an Ingress Controller](http://clouddocs.f5.com/containers/latest/kubernetes/kctlr-k8s-ingress-ctlr.html) in Kubernetes or OpenShift. 
 
-Note that your [F5 k8s-bigip-ctlr](https://github.com/F5Networks/k8s-bigip-ctlr) should already be running in your cluster.
+### Prereqisites
 
-# Suggested Use
+- Install [Helm with Tiller](https://docs.helm.sh/using_helm/#installing-helm) on your cluster with appropriate permissions.
+- Deploy the F5 BIG-IP Controller in your cluster. You can use a separate chart - the [f5-bigip-ctlr chart](https://github.com/F5Networks/charts/tree/master/src/stable/f5-bigip-ctlr) - to deploy the controller, or you can deploy it [manually](http://clouddocs.f5.com/containers/latest/kubernetes/kctlr-app-install.html). 
```

Comment:
You can use the [f5-bigip-ctlr](...) chart to deploy the Controller or deploy it [manually](...).

Reply:
Done.